### DISCUSSION TITLE
Customize YAML and Markdown save criteria

### DISCUSSION THREAD

**Discussion by lorensansol** [StaticJsCMS/static-cms#974] [Q&A]
Hello! I would like to customize the criteria with which to save YAML and Markdown files:

- Literal text blocks instead of folded ones
- Text strings preferably without quotes, but with single quotes if necessary
- Single line text strings without line breaks (as if the width of the document were infinite)
- One less level of indentation in arrays

Would it be possible to do this online and save all compilation?

**Comment by KaneFreeman** [comment]
I should be able to expose the options for the yaml formatter, which should give you more control over how the files are written. I've created an issue for this #976 , and will target adding this to `v4`.
